package.json

```diff
@@ -15,51 +15,52 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "~14.1.0",
-    "@angular/cdk": "^14.1.0",
+    "@angular/cdk": "~14.1.0",
     "@angular/common": "~14.1.0",
     "@angular/compiler": "~14.1.0",
     "@angular/core": "~14.1.0",
-    "@angular/flex-layout": "^14.0.0-beta.40",
+    "@angular/flex-layout": "~14.0.0-beta.40",
     "@angular/forms": "~14.1.0",
-    "@angular/material": "^14.1.0",
+    "@angular/material": "~14.1.0",
     "@angular/platform-browser": "~14.1.0",
     "@angular/platform-browser-dynamic": "~14.1.0",
     "@angular/router": "~14.1.0",
-    "@angular/youtube-player": "^14.1.0",
-    "@auth0/angular-jwt": "^5.1.0",
-    "@swimlane/ngx-charts": "^20.1.2",
-    "angular-typing-animator": "0.0.9",
-    "ngx-csv": "^0.3.2",
-    "ngx-markdown": "^14.0.1",
-    "rxjs": "^7.5.6",
-    "tslib": "^2.4.0",
+    "@angular/youtube-player": "~14.1.0",
+    "@auth0/angular-jwt": "~5.1.0",
+    "@swimlane/ngx-charts": "~20.1.2",
+    "@types/file-saver": "^2.0.5",
+    "file-saver": "^2.0.5",
```

Comment:
Revert the changes to `package` and `package-lock` as we do not need this dependency in production.

Reply:
Try the following:
- On your local `development` branch do `git pull` to get the latest version of dev.
- `git checkout` your local `learning-observer` branch.
- `git pull` to ensure the remote `learning-observer` branch and the local are aligned.
- `git merge development` to merge any changes from development to your local branch.
- `git push` to send all changes from `development` to the remote `learning-observer` branch.